### PR TITLE
Work

### PR DESCRIPTION
**Pull request by rodionov-v**
Was added Stack and Queue functionality

### CODE REVIEW

File: Stack/Program.cs [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+queue.Push(12);
+queue.Push(44);
+queue.Push("Data");
+queue.Push("Data1");
+
+Console.WriteLine(queue.Pop());
+Console.WriteLine(queue.Pop());
+Console.WriteLine(queue.Pop());
+queue.Clear();
+Console.WriteLine(queue.Pop());
```
Comment (vctrro) [inline Stack/Program.cs]: Codestyle. Requares new line at the end

File: Stack/Cell.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
```
Comment (vctrro) [inline Stack/Cell.cs]: Codestyle. Requires new line between using and namespace blocks

**Review by vctrro — CHANGES_REQUESTED** [review]
Check the review comments and fix them

File: Stack/Cell.cs (hunk 2 above)
Comment (rodionov-v) [reply Stack/Cell.cs]:
```suggestion
﻿using System;
﻿
```

**Review by vctrro — APPROVED** [review]
Change approved

File: Stack/Program.cs (hunk 1 above)
Comment (rodionov-v) [reply Stack/Program.cs]:
```suggestion
Console.WriteLine(queue.Pop());

```

Comment (vctrro) [inline Stack/Program.cs]: Codestyle. Requares new line at the end

**Review by vctrro — CHANGES_REQUESTED** [review]

File: Stack/Program.cs (hunk 1 above)
Comment (rodionov-v) [reply Stack/Program.cs]:
```suggestion
Console.WriteLine(queue.Pop());

```

**Pull request merged**